var/spack/repos/builtin/packages/expat/package.py

```diff
@@ -31,5 +31,24 @@ class Expat(AutotoolsPackage):
     homepage = "http://expat.sourceforge.net/"
     url      = "https://sourceforge.net/projects/expat/files/expat/2.2.2/expat-2.2.2.tar.bz2"
 
+    # Version 2.2.2 introduced a requirement for a high quality
```

Comment:
I think that there were some get_random knobs in 2.2.1 ([here](https://github.com/libexpat/libexpat/blob/c4446687cfc6c5fd7f6371aeaf24c69402a3589e/expat/lib/xmlparse.c#L698), if I've navigated the release right) but the code that makes the build fail came in for 2.2.2 ([here](https://github.com/libexpat/libexpat/commit/ff0207e6076e9828e536b8d9cd45c9c92069b895), ditto) and [here(changelog)](https://github.com/libexpat/libexpat/blob/master/expat/Changes#L40).